Userland/SampleCodeModule/userstdlib.c

```diff
@@ -57,6 +57,15 @@ int atoi(const char* str) {
     return neg * answer;
 }
 
+void sleep(unsigned long millis) {
+	int start = sys_millis();
+    int current;
+    do{
+         current = sys_millis();
+         sys_yield();
+    }while (current - start < millis);
+}
```

Comment:
`sys_millis()` should be called after `sys_yield()`, otherwise you're comparing an older value

```suggestion
    unsigned long start = sys_millis();
    do {
         sys_yield();
    } while (sys_millis() - start < millis);
}
```